RebelAllianceBank/Classes/FileHandler.cs

```diff
@@ -0,0 +1,153 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using RebelAllianceBank.Interfaces;
+
+namespace RebelAllianceBank.Classes
+{
+    internal class FileHandler
+    {
+        public List<IUser> ReadUsersFromFile()
+        {
+            List<IUser> savedList = new List<IUser>();
+            string filePath = "..\\..\\..\\users.txt";
+            if (File.Exists(filePath))
+            {
+                using (StreamReader sr = new StreamReader(filePath))
+                {
+                    string read;
+                    while ((read = sr.ReadLine()) != null)
+                    {
+                        string[] row = read.Split('-');
+                        IUser user = StoredUser(row);
+                        if (user != null)
+                        {
+                            savedList.Add(user);
+                        }
+                    }
+                }
+            }
+            return savedList;
+        }
```

Comment:
This method can be simplify to a more general method.

Reply:
Merged methods to generic